proto/minder/v1/minder.proto

```diff
@@ -2455,3 +2455,72 @@ message Provider {
     // This is an output-only field. It may be: "set", "unset", "not_applicable".
     string credentials_state = 9;
 }
+
+message EvaluationHistory {
+    // entity contains details of the entity which was evaluated.
+    EvaluationHistoryEntity entity = 1;
+
+    // rule contains details of the rule which the entity was evaluated against.
+    EvaluationHistoryRule rule = 2;
+
+    // status contains the evaluation status.
+    EvaluationHistoryStatus status = 3;
+
+    // alert contains details of the alerts for this evaluation.
+    EvaluationHistoryAlert alert = 4;
+
+    // remediation contains details of the remediation for this evaluation.
+    EvaluationHistoryRemediation remediation = 5;
+}
+
+message EvaluationHistoryEntity {
```

Comment:
Since these messages are only used inside `EvaluationHistory`, have you considered defining them there? (You could still access them outside with a qualified name, see https://protobuf.dev/programming-guides/proto3/#nested)

Reply:
I like the idea but don't fully grasp all implications.
Those messages are not yet used in the application, I would move them in a separate PR.